### PR TITLE
SAIC-543: Cinder volume migration

### PR DESCRIPTION
**Pull request by aguzikova**

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.storage.cinder_storage
cloudferrylib/os/storage/cinder_storage.py:95: [W0221(arguments-differ), CinderStorage.deploy] Arguments number differs from overridden 'deploy' method
cloudferrylib/os/storage/cinder_storage.py:153: [W1201(logging-not-lazy), CinderStorage.finish] Specify string format arguments as logging function parameters
cloudferrylib/os/storage/cinder_storage.py:252: [W0613(unused-argument), CinderStorage.convert_snapshot] Unused argument 'cloud'
************* Module cloudferrylib.os.storage.cinder_database
cloudferrylib/os/storage/cinder_database.py:223: [W1202(logging-format-interpolation), CinderStorage.deploy_data_to_table.filter_data] Use % formatting in logging functions but pass the % parameters as arguments
************* Module cloudferrylib.utils.remote_runner
cloudferrylib/utils/remote_runner.py:31: [W0621(redefined-outer-name), RemoteRunner.__init__] Redefining name 'sudo' from outer scope (line 15)
************* Module tests.cloudferrylib.os.actions.test_cinder_database_manipulation
tests/cloudferrylib/os/actions/test_cinder_database_manipulation.py:31: [W0613(unused-argument), volume_size] Unused argument 'vol_fil'
tests/cloudferrylib/os/actions/test_cinder_database_manipulation.py:35: [W0613(unused-argument), free_space] Unused argument 'path'
************* Module tests.generate_load
devlab/tests/generate_load.py:7: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:9: [F0401(import-error), ] Unable to import 'filtering_utils'
devlab/tests/generate_load.py:36: [W0621(redefined-outer-name), clean_if_exists.wrapper] Redefining name 'args' from outer scope (line 1188)
devlab/tests/generate_load.py:971: [W0120(useless-else-on-loop), CleanEnv.clean_vms.wait_until_vms_all_deleted] Else clause on loop without a break statement
devlab/tests/generate_load.py:977: [W0106(expression-not-assigned), CleanEnv.clean_vms] Expression "[vms.append(vm) for vm in self.config.vms_from_volumes]" is assigned to nothing
devlab/tests/generate_load.py:1122: [W0141(bad-builtin), CleanEnv.clean_keypairs.delete_user_keypairs] Used builtin function 'map'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cfglib.py [hunk 1]
```diff
@@ -170,6 +170,10 @@
                      'migration process', default=False),
     cfg.BoolOpt('hide_ssl_warnings', default=False,
                 help="Don't show ssl warnings"),
+    cfg.StrOpt('src_cinder_conf', default='/etc/cinder/cinder.conf',
```
Comment (roman-verchikov) [inline cfglib.py]: We already have `src_storage_opts` and `dst_storage_opts` for keeping all cinder-related options. Please move these there.

File: cloudferrylib/os/actions/cinder_database_manipulation.py [hunk 2]
```diff
@@ -47,18 +114,376 @@ def run(self, *args, **kwargs):
...
+            cmd = (
+                "for exp in "
+                "$(awk -F= '/nfs_shares_config/ "
+                "{print $2; exit}' '%s' | xargs cat); "
+                "do mount | "
+                "awk '{if (match($1, \"^'$exp'$\") && $3 ~ \"cinder\") "
+                "{print $3}}'; done"
+            ) % self.cinder_conf[position]
+        res = self._run_cmd(self.cloud[position], cmd)
+        return res if isinstance(res, list) else [res]
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/cinder_database_manipulation.py]: `mount_dirs()` looks like copy-paste of `_mount_point()`

**Pull request closed without merging**